server/cli.js

```diff
@@ -113,7 +130,33 @@ if (options) {
       writeToFile(report, options);
     });
   } else if (argv.report === "attendance") {
-    getAttendanceReport(options.format, options.startDate, options.endDate, argv.minStudents).then((report) => {
+    let promise = getAttendanceReport(
+      options.format, options.startdate, options.enddate);
```

Comment:
I think you forgot `argv.minStudents`.

Reply:
It's in there now. I ran the code from this branch and used minStudents and it worked.